plugins/ivanti_service_manager/unit_test/test_add_note.py

```diff
@@ -0,0 +1,47 @@
+import sys
+import os
+import json
+
+sys.path.append(os.path.abspath("../"))
+
+from unittest import TestCase
+from unittest.mock import patch
+from icon_ivanti_service_manager.actions.add_note.schema import Input
+from unit_test.util import Util
+from unit_test.mock import mock_request
+from icon_ivanti_service_manager.actions.add_note import AddNote
+
+
+class TestAddNote(TestCase):
+    @classmethod
+    def setUpClass(cls) -> None:
+        cls.params = {
+            "category": "category",
+            "notes": "notes",
+            "source": "source",
+            "summary": "summary",
+            "incident_number_good": 12345,
+            "incident_number_bad": 54321,
+        }
```

Comment:
I suggest moving that parameters to a const variable instead of putting that to `setUpClass`